package.json

```diff
@@ -1,47 +1,53 @@
 {
-  "name": "dockerode",
-  "description": "Docker Remote API module.",
-  "version": "3.3.1",
-  "author": "Pedro Dias <petermdias@gmail.com>",
-  "maintainers": [
-    "apocas <petermdias@gmail.com>"
-  ],
+  "name": "sim-pipe-simulation-controller",
+  "version": "0.0.1",
+  "description": "Simulations and analytics for DataCloud pipelines.",
+  "exports": "./dist/index.js",
+  "type": "module",
+  "scripts": {
+    "start": "node --experimental-specifier-resolution=node --loader ts-node/esm/transpile-only ./src/index.ts",
```

Comment:
This beautiful command is what is required to run TypeScript NodeJS code using ESM modules. I'm sorry.

I would recommend using the VsCode debugger instead of the command line to run the code.

Reply:
So to run the application outside vscode you can run one of the two commands:

```bash
npm run start
```

```bash
node --experimental-specifier-resolution=node --loader ts-node/esm/transpile-only ./src/index.ts
```